docs/_sources/quick.rst.txt

```diff
@@ -0,0 +1,102 @@
+.. _quick:
+
+Quick Start 
+===========
+
+Calculate Fairness Metrics:
```

Comment:
We mentioned adding anchors to the quick start examples before, we should add them.